pkg/syncer/syncer.go

```diff
@@ -218,6 +219,26 @@ func StartSyncer(ctx context.Context, cfg *SyncerConfig, numSyncerThreads int, i
 
 	go specSyncer.Start(ctx, numSyncerThreads)
 	go statusSyncer.Start(ctx, numSyncerThreads)
+	// connect to create the reverse tunnels
+	var (
+		initBackoff   = 5 * time.Second
+		maxBackoff    = 5 * time.Minute
+		resetDuration = 1 * time.Minute
+		backoffFactor = 2.0
+		jitter        = 1.0
+		clock         = &clock.RealClock{}
+		sliding       = true
+	)
+
+	backoffMgr := wait.NewExponentialBackoffManager(initBackoff, maxBackoff, resetDuration, backoffFactor, jitter, clock)
```

Comment:
there is a default backoff. Isn't good enough?

Reply:
and if not we should rather copy that down into some func. Now the StartSyncer func gets crowded too much by these details.

Reply:
the retry.DefaultBackoff? that is too quick for these things

Reply:
or do we have some Default in other place?

Reply:
this different backoff is fine. Just move it out of this function.

Reply:
done